### ISSUE TITLE
Snagit and Camtasia both get an error on download

### ISSUE THREAD

**Issue by beatlemike** [autopkg/bkerns-recipes#21]
```
<urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain (_ssl.c:1076)>
Failed.
```

**Comment by WaveBirdDash** [comment]
Seeing this same error, but only in AutoPkg 2.x (with Python 3); AutoPkg 1.4.1 is running the recipe successfully. The error is specifically in `TechSmithURLProvider`, so presumably that processor will need to be updated for Python 3 compatibility (in #20 maybe?).

**Comment by beatlemike** [comment]
Yes only an error in autopkg 2.x

**Comment by kernsb** [comment]
The fix in #20 was merged, is this issue still happening?

**Comment by flammable** [comment]
Thanks, it's fixed for me!

**Comment by beatlemike** [comment]
Yep meant to mark this yesterday, didn't get to verify until last night. All working!

**Comment by kernsb** [comment]
Fixed with merge from #20
